Controllers/BranchesController.cs

```diff
@@ -39,7 +39,7 @@ public async Task<IActionResult> Index()
             return View(branchesVM);
         }
 
-        // GET: Entities
+        // GET: Branches by Entity Id
```

Comment:
Idea behind this comment was point out route to endpoint. 
This is nothing important, but I would remove this comment or typed:
`// GET: Entity/{id}
As you know I am allergic to comments in code :P

Reply:
look to line 61